.vib/apache/goss/vars.yaml

```diff
@@ -0,0 +1,49 @@
+components:
+  binaries:
+    - httpd
+    - gosu
+    - render-template
+  files:
+    - /opt/bitnami/apache/conf/modsecurity.conf
+    - /opt/bitnami/apache/conf/unicode.mapping
+filesystem:
+  - mode: "0775"
+    directories:
+      - /opt/bitnami/apache/var/run
+      - /opt/bitnami/apache/conf
+      - /opt/bitnami/apache/logs
+      - /opt/bitnami/apache/conf/vhosts
+      - /opt/bitnami/apache/conf/vhosts/htaccess
+      - /opt/bitnami/apache/htdocs
+modules:
+  loaded:
+    # Explicitly enabled in postunpack
+    - deflate
+    - negotiation
+    - proxy
+    - proxy_http
+    - rewrite
+    - slotmem_shm
+    - socache_shmcb
+    - ssl
+    - status
+    - version
+    # Explicitly set at compilation time
+    - mpm_prefork
+  disabled:
+    # Explicitly disabled in postunpack
+    - http2
+    - proxy_hcheck
+  extra:
+    # Not enabled, but compiled
+    - mpm_event
+    - mpm_prefork
+    - mpm_worker
+    - http2
+    - ldap
+    - authnz_ldap
+    - security3
+root_dir: /opt/bitnami
+version:
+  bin_name: apachectl
+  flag: -v
```

Comment:
Not used

Reply:
Good catch

Reply:
These entries are indeed used, but not in the `apache.yaml` file directly.

https://github.com/bitnami/containers/blob/dfed25e88341e581baeae622fb2a85111941df83/.vib/common/goss/templates/check-app-version.yaml#L1-L6